src/olympia/lib/settings_base.py

```diff
@@ -1833,8 +1830,31 @@ def get_redis_settings(uri):
     'ORDERING_PARAM': 'sort',
 }
 
+
+def get_raven_release():
+    version_json = os.path.join(ROOT, 'version.json')
+    version = None
+
+    if os.path.exists(version_json):
+        try:
+            with open(version_json, 'rb') as fobj:
+                version = json.loads(fobj.read())['version']
+        except (IOError, KeyError):
+            version = None
+
+    if version is None or version == 'origin/master':
+        version = raven.fetch_git_sha(ROOT)
```

Comment:
`version.json` will always exist for builds by circleci (or our circleci build process is broken), and dev, stage and prod are always deployed from circleci builds. So should we get the commit out of `version.json` instead if `version` isn't set?

Reply:
Indeed, I defaulted to fetch it from version.json as well in 5abf6ff967

Reply:
Indeed, I defaulted to fetch it from version.json as well in 5abf6ff967